libs/design/src/core/compactable/compactable.directive.ts

```diff
@@ -18,7 +20,7 @@ import {
  * <div daffCompactable [compact]="isCompact">Content goes here</div>
  * ```
  *
- * @example Implementing it as an Angular host directive
+ * Implementing it as an Angular host directive
```

Comment:
I think this was correct before

Reply:
So I need to double tag `@example` even if everything under the first `@example` should be rendered in the body? Or is this so that the two examples can be split up and rendered separately?

Reply:
So two examples can be rendered separately.

Reply:
updated